create_jenkins_job.py

```diff
@@ -216,9 +216,9 @@ def create_job(os_name, job_name, template_file, additional_dict):
             packaging_label_expression = 'macos &amp;&amp; mojave'
 
         # configure a manual version of the packaging job
-        ignore_rmw_default_packaging = {'rmw_opensplice_cpp'}
+        ignore_rmw_default_packaging = { 'rmw_opensplice_cpp' }
```

Comment:
I accidentally introduced spaces between the braces. If needed I can fix it.

Reply:
I just went ahead and fixed it, no worries.